alembic/versions/fe7656c21eaa_add_remainder_of_database_models.py

```diff
@@ -0,0 +1,67 @@
+"""add remainder of database models
+
+Revision ID: fe7656c21eaa
+Revises: 5344fc3a3d3f
+Create Date: 2018-08-01 15:44:29.298131
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fe7656c21eaa'
+down_revision = '5344fc3a3d3f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'sources',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('uuid', sa.String(length=36), nullable=False),
+        sa.Column('journalist_designation', sa.String(length=255),
+                  nullable=False),
+        sa.Column('is_flagged', sa.Boolean(), nullable=True),
+        sa.Column('public_key', sa.String(length=10000), nullable=True),
+        sa.Column('interaction_count', sa.Integer(), nullable=False),
+        sa.Column('is_starred', sa.Boolean(), nullable=True),
+        sa.Column('last_updated', sa.DateTime(), nullable=True),
```

Comment:
A pattern I've generally used is to keep two timestamps on records like this: a `created_at` and an `updated_at`. Not sure in this case it matters, but in previous projects it's been nice to know both in different circumstances.